packages/react-native/ReactCommon/react/renderer/graphics/ValueUnit.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+namespace facebook::react {
+
+enum class UnitType {
```

Comment:
Why do we need undefined?

Reply:
To represent initial/undefined value? Also useful to check the value is set. Check isSet in TransformOrigin struct.